src/rules.mk

```diff
@@ -310,6 +310,19 @@ src_erf_LDADD = \
     $(LDADD) \
     $(EMPTY)
 
+bin_PROGRAMS += src/xml2gff
+src_xml2gff_SOURCES = \
+    src/xml2gff.cpp \
+    src/util.cpp \
+    $(EMPTY)
+src_xml2gff_LDADD = \
+    src/xml/libxml.la \
+    src/aurora/libaurora.la \
+    src/common/libcommon.la \
+    src/version/libversion.la \
+    $(LDADD) \
+    $(EMPTY)
+
```

Comment:
Can you add the binary (both Unix and Windows) to the .gitignore file too, please?

(Also, a manpage would be great as well)